python/llm/example/CPU/Application/auto_comment/README.md

```diff
@@ -0,0 +1,116 @@
+# Auto Comment
+
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Qwen models to automatically generate comments for code files. For illustration purposes, we utilize the [Qwen/Qwen-VL-7B](https://huggingface.co/Qwen/Qwen-VL-7B) as a reference Qwen model.
```

Comment:
> In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Qwen models to automatically generate comments for code files.

change to

In this directory, you will find an example of using LLM to automatically generate comments for code files. This example is adapted from \[QWen Example\](add link to reference code).  We utilize the [Qwen/Qwen-VL-7B](https://huggingface.co/Qwen/Qwen-VL-7B) optimized w/ BigDL-LLM INT4 for illustration.